UnityProject/Assets/Scripts/HealthV2/Living/BodyPartMutations.cs

```diff
@@ -38,6 +38,43 @@ public static MutationRoundData GetMutationRoundData(MutationSO Mutation)
 		return MutationVariants[Mutation];
 	}
 
+	public override void Awake()
+	{
+		base.Awake();
+		RelatedPart.OnDamageChangeBy += OnDMGMutationCheck;
+	}
+
+
+	public void OnDMGMutationCheck(AttackType attackType,DamageType damageType, float amount)
+	{
+		if (damageType == DamageType.Clone)
+		{
+			//Range = -999999 to 9999999
+			if (amount > 0)
+			{
+				//Range = 0 to 9999999
+				amount = Mathf.Clamp(amount, 0, 100);
+				//Range = 0 to 100
+				//Percentage 100 = 10
+
+				var RNG= Random.Range(0, 1000);
+				if (amount >= RNG)
```

Comment:
amount can only be ``100`` at most. Why make the range from 0 to 1000?

Reply:
so 100 = 10% chance of happening